src/device-registry/controllers/create-event.js

```diff
@@ -148,6 +148,87 @@ const createEvent = {
     }
   },
 
+  latestFromBigQuery: async (req, res) => {
```

Comment:
new controller logic to handle the retrieval of latest measurements. 
Might need review since it introduces code duplication in our code base, need to utilise some SOLID principles on this.